src/logic/dictionaryUtils.ts

```diff
@@ -3,6 +3,28 @@ import { Commander } from "../types/domain/Commander";
 import { Match } from "../types/domain/Match";
 import { Player } from "../types/domain/Player";
 
+/**
+ * Given a collection of Matches, filter out games that don't have the desired number of players
+ * "Seats" refers to the desired number of seats in matches to keep (usually 4)
+ * @param matches 
+ * @param seats 
+ * @returns 
+ */
+
+export function filterMatchesBySeatCount(matches: Match[], seats: number): Match[] {
```

Comment:
nit: let's name this as filterMatchesByPlayerCount

The concept of a "Seat" doesn't exist, so introducing a new term to the source isn't something I want to do yet. Good name though, and we can think about it in the future.

Reply:
I deliberately chose to use the concept of "seat" to distinguish it from edge cases where multiple people might be playing the same deck (2H). Thoughts?

Reply:
Sorry, not "playing the same deck" but "playing on the same team".